activity.js

```diff
@@ -18,13 +18,17 @@ class Activity {
       this.seconds = 59;
     } else if (this.minutes === 0 && this.seconds === 0) {
       this.completed = true;
-      return;
-    }
+      startTimerButton.innerText = `COMPLETE!`
+      showElement(logActivityButton);
+      showElement(newActivityButton);
+      return 
+    };
+
```

Comment:
Did we decide to return the innerText? Or do that in two lines? Wasn't it faster with only returning it in one line? I can't remember.

Reply:
Also, @Cody-Durham, I wanted to work out the fix for making that complete text showing up on zero instead of a second after zero. That's the block I slacked. Maybe we should include that on this branch. We can walk through it together when we get together today, or if we're both around earlier than that.